src/site/layouts/campaign_landing_page.drupal.liquid

```diff
@@ -649,7 +649,9 @@
         </div>
       </div>
     </div>
-    {% include "src/site/components/up_to_top_button.html" %}
+
+    <!-- Back to Top -->
+    {% include "src/site/components/up_to_top_button.html" with isCampaignLandingPage = true %}
```

Comment:
can we rename `isCampaignLandingPage` to something like `fullWidth`, in case this comes up again?

Reply:
I was thinking about doing that initially, but then if you look at it in desktop it doesn't really _mean_ full width actually since the up to top button doesn't actually go to the right side of the viewport. So I figured if this is really just an edge case with CLP and its markup, then a variable name that was this specific probably would be clearest for now. What do you think?